processing/data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, date
+
+import dateutil
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMossoroSpider(BaseGazetteSpider):
+
+    name = "rn_mossoro"
+    allowed_domains = ["jom.prefeiturademossoro.com.br"]
+
+    TERRITORY_ID = "2408003"
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        self.start_date = date.today().replace(day=1)
```

Comment:
would be better to set this value only if none right? and also, only after the super statement I think

Reply:
That's the opposite, actually. The super class only touches `self.start_date` if the spider args are defined, so the idea here is:
1- Set default values
2- Override the default value if the user defined something valid.